src/main/webapp/map.html

```diff
@@ -44,31 +43,37 @@
         marker.addListener('click', function() {
             infoWindow.open(map, marker);
         });
-     }
+     }*/
+    function createSightingsMap(){
+      fetch('/map-data').then(function(response) {
+        return response.json();
+      }).then((Sightings) => {
 
+        const map = new google.maps.Map(document.getElementById('map'), {
+          center: {lat: 24.943325, lng: 121.297187},
```

Comment:
Let's add a comment for what location this lat/lng represents.

Reply:
okay. I have added a comment for the location